src/schema/v2/artist/index.ts

```diff
@@ -272,6 +272,7 @@ export const ArtistType = new GraphQLObjectType<any, ResolverContext>({
                     hasPreviousPage: page > 1,
                     hasNextPage: page < totalPages,
                   },
+                  artist_id: _id,
```

Comment:
#minor can we make `artist_id` 🐫 case? 

#veryminor It look initially a bit strange that `artist_id` and `pageInfo` are under the same `{}` but `totalCount` is under a different object but I guess `merge` takes care of that.

Reply:
`artist_id` in every other position in this file isn't camel case. I'd rather leave it to match the current pattern and update it all at once in the future.